P0009/P0009.md

```diff
@@ -1935,8 +1922,8 @@ template<class... Indices>
 
 [3]{.pnum} *Preconditions:* `static_cast<size_type>(i)` is a multidimensional index in $extents\_$ ([mdspan.terms]). 
 
-[4]{.pnum} *Effects:* Let `P` be the parameter pack such that
-            `is_same_v<make_index_sequence<sizeof...(Indices)>, index_sequence<P...>>` is `true`.
+[4]{.pnum} *Effects:* Let `P` be a parameter pack such that
```

Comment:
LWG seemed to prefer a different way of saying that two types are the same, other than `is_same_v`.  Would that way apply here as well?

Reply:
They did not say it here because the language was stuff like "where something::some_type denotes TYPE". But here we are saying something about P which is not a type, but a pack so having this conditional seems probably better.